chirp/src/chirp_tool.c

```diff
@@ -446,6 +446,26 @@ static INT64_T do_getacl(int argc, char **argv)
 	return chirp_reli_getacl(current_host, full_path, print_one_acl, stdout, stoptime);
 }
 
+static INT64_T do_link(int argc, char **argv)
+{
+	INT64_T result;
+	char old_full_path[CHIRP_PATH_MAX];
+	char new_full_path[CHIRP_PATH_MAX];	
+	int sym = (argc == 4 && strcmp(argv[1], "-s") == 0);
+
+	if(sym) {
+		complete_remote_path(argv[2], old_full_path);
```

Comment:
`argv[2]` is the target of the symbolic link and doesn't necessarily have to be a valid path. It should not be "fixed" by `complete_remote_path`. Consider:

````
chirp host:port <<EOF
chdir a/directory
ln -s /an/absolute/path ./foo
EOF
````

This should create a symbolic link `/a/directory/foo` that points to `/an/absolute/path`.